packages/react-storage/src/components/StorageBrowser/composables/Search.tsx

```diff
@@ -68,23 +65,11 @@ export const Search = ({
       </Field>
       <ButtonElement
         className={`${BLOCK_NAME}__submit-button`}
-        onClick={() => onSearch?.(term, subfoldersIncluded)}
+        onClick={() => onSearch?.()}
```

Comment:
Should be able to just pass `onSearch` idiomatically, same with `onSearchClear`?